llvm/tools/llvm-split/llvm-split.cpp

```diff
@@ -53,6 +53,12 @@ static cl::opt<bool>
                    cl::desc("Split without externalizing locals"),
                    cl::cat(SplitCategory));
 
+static cl::opt<bool>
+    TryToAvoidEmptyModules("avoid-empty-modules", cl::Prefix, cl::init(false),
```

Comment:
I am wondering can we make this option default? We could still introduce a flag that opt-outs?

Reply:
Can be done but I'd prefer changing the default behavior to be a separate change so that the current change would not need to be reverted if someone relies on the current behavior.